spyder/plugins/ipythonconsole/widgets/client.py

```diff
@@ -379,13 +379,17 @@ def get_kernel(self):
 
     def get_options_menu(self):
         """Return options menu"""
+        interrupt_action = create_action(self, _("Interrupt kernel"),
+                                     icon=self.stop_icon,
+                                     triggered=self.stop_button_click_handler)
+               
         reset_action = create_action(self, _("Remove all variables"),
                                      icon=ima.icon('editdelete'),
                                      triggered=self.reset_namespace)
 
         env_action = create_action(
                         self,
-                        _("Show environment variables"),
+                        _("Show env vars"),

```

Comment:
``"Show environment variables"`` is clearer to those who may not know the meaning of the abbreviations (non-natives, newer users, etc) and still shorter than ``Connect to an existing kernel`` so it doesn't make the menu any wider anyway.